src/profile.c

```diff
@@ -83,9 +83,8 @@
 **     we provide -P (profiling) and -c (code coverage) options to the GAP
 **     executable so the user can start before code loading starts
 **
-**  3) Operating at just a line basis can sometimes be too course. We can
-**     see this when we use ActivateProfileColour. However, without some
-**     serious additional overheads, I can't see how to provide this
+**  3) Operating at just a line basis can sometimes be too course. However,
+**     without some serious additional overheads, I can't see how to provide this
```

Comment:
There is another reference to colo(u)r in line 66. Also, the variable `ColouringOutput` is not actually removed. 

I am going to update this PR to fix this and my other remarks.